.github/workflows/node_integration_tests.yml

```diff
@@ -59,18 +76,18 @@ jobs:
         shell: bash
         run: |
           nohup npm run start-feature &
-          npm --prefix integration_tests clean-install --no-audit &&
-            npm --prefix integration_tests run build
           nohup npm run start-feature-test-app &
```

Comment:
this 1 line is the main difference from the upstream workflow, but also the wait-for-app-to-be-ready mechanism is more careful